piker/brokers/kucoin.py

```diff
@@ -0,0 +1,668 @@
+# Copyright (C) Jared Goldman (in stewardship for pikers)
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+'''
+Kucoin broker backend
+
+'''
+
+from random import randint
+from typing import Any, Callable, Optional, Literal, AsyncGenerator
```

Comment:
as per other code,i prefer the multi-line version:

```python
from typing import (
	Any,
	Callable,
	Optional,
	Literal,
	AsyncGenerator,
)
```

Reply:
Done